lib/public/views/lhcPeriods/Overview/LhcPeriodsOverviewModel.js

```diff
@@ -32,6 +37,10 @@ export class LhcPeriodsOverviewModel extends Observable {
         this._pagination = new PaginationModel();
         this._pagination.observe(() => this.fetchLhcPeriods());
         this._pagination.itemsPerPageSelector$.observe(() => this.notify());
+        this._pagination.provideDefaultItemsPerPage(estimateDisplayableRowsCount(
+            TABLEROW_HEIGHT,
+            PAGE_USED_HEIGHT,
+        ), { silent: true });
```

Comment:
Why diverge from how it is done on other components?
If your way is better, then if it do not imply too much change please do it everywhere